packages/react-core/src/components/Menu/Menu.tsx

```diff
@@ -156,20 +156,22 @@ class MenuBase extends React.Component<MenuProps, MenuState> {
       this.setState({ transitionMoveTarget: null });
     } else {
       const nextMenu = current.querySelector('#' + this.props.activeMenu) || current || null;
-      const nextMenuChildren = Array.from(nextMenu.getElementsByTagName('UL')[0].children);
+      const nextMenuLists = nextMenu.getElementsByTagName('UL');
+      if (nextMenuLists.length > 0) {
```

Comment:
I'd probably do the opposite here and just `return` if empty just to make it more clear logic ends and to avoid a level of nesting.